### DISCUSSION TITLE
What's the purpose of "examples" attribute of Field in Pydantic v2?

### DISCUSSION THREAD

**Discussion by MasterGroosha** [pydantic/pydantic#9501] [Pydantic V2]
From the documentation (https://docs.pydantic.dev/latest/concepts/json_schema/#field-level-customization) it seems that instead of "example" there is now "examples" attribute of Field which accepts list of values. However, if I specify the list, only the first one is shown in Swagger and ReDoc.
What's the purpose of "examples"? Can I continue using "example" instead?

**Comment by guillaume-alvarez** [comment]
The 'examples' keyword appeared only in OpenAPI 3.1.
Before that you could only have one 'example'.
